doc/datasets/index.rst

```diff
@@ -262,6 +277,8 @@ Generators for decomposition
 Loading other datasets
 ======================
 
+This section gathers different tools to load other kinds of datasets.
```

Comment:
This sentence brings very little information I would rather remove it to avoid diluting the actual information with boilerplate English connection sentences.